### ISSUE TITLE
Focus doesn't recognise user installed certificates for https

### ISSUE THREAD

**Issue by samuel-emrys** [mozilla-mobile/focus-android#3521]
## Steps to reproduce
1. Configure SSL connections in pfSense webConfigurator
2. Export root CA to android device and install as user certificate
3. Open Focus and navigate to pfSense web portal, i.e. https://192.168.0.1/

### Expected behavior
User certificate is identified as valid, and an SSL connection is opened to pfSense web portal

### Actual behavior
Redirected to error page: "Secure connection failed"
Tapping the "!" button gives "Insecure connection"

### Device information

* Android device: 8.0.0
* Focus version: 6.1.1
* Device: Samsung Galaxy S7 edge

Not sure if this belongs as a bug report or feature request, but I've been unable to get focus to recognise a self-signed certificate for my pfSense web portal. This same procedure was able to achieve a secure connection to the same page using Chrome 69.0.3497.100

**Comment by npark-mozilla** [comment]
@sv-ohorvath could you repro this on 7.0 and see whether it still happens?

**Comment by sv-ohorvath** [comment]
Sorry, I personally don't have knowledge in this area ( also asked around for help but without success) and can't verify this. @samuel-emrys, maybe you can check on the latest Focus version: https://github.com/mozilla-mobile/focus-android/releases/tag/v7.0-RC10 and let us know if you're still having issues? Thank you!

**Comment by samuel-emrys** [comment]
didn't get a chance to build 7.0-RC10 from source myself, but I've just tried with 7.0.11 that dropped yesterday and still not working. To add to the reproduction steps:
1. Install pfSense in a VM: https://www.ceos3c.com/pfsense/install-pfsense-2-4-4/
2. Configure SSL connections in pfSense webConfigurator: https://www.ceos3c.com/pfsense/pfsense-generate-ssl-certificate-https-pfsense/

This goes through the process of creating the CA's and individual certificate, as well as how to export the root CA.

3. Install the root CA on android device
3.1 Export the root CA from pfSense
3.2 Downloading root CA on android device should automatically install (I just emailed it to myself)

Note: It will automatically download as a .crt file; I'm not sure if installation of .crt files is supported by android. I converted it to a .pem file using openssl:

`openssl x509 -in mycert.crt -out mycert.pem -outform PEM`

Installation can be verified in Settings > Lock screen and security > other security settings > User certificates

4. Navigate to pfSense web portal, i.e. https://192.168.0.1/

As mentioned previously, Chrome recognises the user certificate at this point, and Focus does not.
